src/i18n/en/nav.ts

```diff
@@ -18,190 +18,105 @@
+
+	{ text: 'Client-side interactivity', header: true, type: 'learn', key: 'clientSide' },
+	{ text: 'UI Frameworks', slug: 'guides/framework-components', key: 'guides/framework-components' },
+	{ text: 'Scripts & Event Handling', slug: 'guides/client-side-scripts', key: 'guides/client-side-scripts' },
+
+
+	{ text: 'Assets', header: true, type: 'learn', key: 'assets' },
 	{ text: 'CSS & Styling', slug: 'guides/styling', key: 'guides/styling' },
 	{ text: 'Images', slug: 'guides/images', key: 'guides/images' },
 	{ text: 'Fonts', slug: 'guides/fonts', key: 'guides/fonts' },
-	{ text: 'Imports', slug: 'guides/imports', key: 'guides/imports' },
-	{ text: 'Endpoints', slug: 'guides/endpoints', key: 'guides/endpoints' },
-	{ text: 'Data Fetching', slug: 'guides/data-fetching', key: 'guides/data-fetching' },
 
-	{
-		text: 'Internationalization',
-		slug: 'guides/internationalization',
-		key: 'guides/internationalization',
-	},
-	{ text: 'Middleware', slug: 'guides/middleware', key: 'guides/middleware' },
-	{ text: 'E-commerce', slug: 'guides/ecommerce', key: 'guides/ecommerce' },
+
+	{ text: 'Maintain your project', header: true, type: 'learn', key: 'maintain' },
+	{ text: 'Upgrade Astro', slug: 'upgrade-astro', key: 'upgrade-astro' },
 	{ text: 'Testing', slug: 'guides/testing', key: 'guides/testing' },
-	{ text: 'Authentication', slug: 'guides/authentication', key: 'guides/authentication' },
 	{ text: 'Troubleshooting', slug: 'guides/troubleshooting', key: 'guides/troubleshooting' },
 
-	{ text: 'Configuration', header: true, type: 'learn', key: 'configuration' },
-	{
-		text: 'The Astro Config File',
-		slug: 'guides/configuring-astro',
-		key: 'guides/configuring-astro',
-	},
-	{ text: 'TypeScript', slug: 'guides/typescript', key: 'guides/typescript' },
-	{ text: 'Import Aliases', slug: 'guides/aliases', key: 'guides/aliases' },
-	{
-		text: 'Environment Variables',
-		slug: 'guides/environment-variables',
-		key: 'guides/environment-variables',
-	},
 
 	{ text: 'Reference', header: true, type: 'api', key: 'reference' },
-	{
-		text: 'Configuration',
-		slug: 'reference/configuration-reference',
-		key: 'reference/configuration-reference',
-	},
-	{ text: 'Runtime API', slug: 'reference/api-reference', key: 'reference/api-reference' },
-	{
-		text: 'Integrations API',
-		slug: 'reference/integrations-reference',
-		key: 'reference/integrations-reference',
-	},
+	{ text: 'Configuration', slug: 'reference/configuration-reference', key: 'reference/configuration-reference' },
+	{ text: 'Astro Runtime API', slug: 'reference/api-reference', key: 'reference/api-reference' },
+	{ text: 'Astro CLI', slug: 'reference/cli-reference', key: 'reference/cli-reference' },
+	{ text: 'Directives Reference', slug: 'reference/directives-reference', key: 'reference/directives-reference' },
+	{ text: 'Error Reference', slug: 'reference/error-reference', key: 'reference/error-reference' },
+
+
+	{ text: 'Other Develoment APIs', header: true, type: 'api', key: 'dev' },
```

Comment:
```suggestion
	{ text: 'Other Astro APIs', header: true, type: 'api', key: 'dev' },
```
Note: Typo here if we keep it as "Development".

I think it would help to make it obvious that this is a section for developer tools.
Perhaps "Astro" APIs, is not correct, though. Maybe "Extend Astro" or "Create an Integration" could also work.